src/pages/Account.vue

```diff
@@ -7,38 +7,43 @@
 
                 <transition name="fade"><div v-if="errorMessage" v-bind:class="[goodFeedback ? 'good message' : 'message' ]">{{ errorMessage }}</div></transition>
 
+                <div v-if="voucherPayments.length > 0">
```

Comment:
Probably something like this but in a computed value:
`(voucherPayments[0]) ? voucherPayments[0].length > 0 : false`